core/server/influx-metrics.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const request = require('request')
+const { promClientJsonToInfluxV2 } = require('./metrics/format-converters')
+const log = require('./log')
+
+module.exports = class InfluxMetrics {
+  constructor(metricInstance, instanceMetadata, config) {
+    this._metricInstance = metricInstance
+    this._instanceMetadata = instanceMetadata
+    this._config = config
+  }
+
+  async registerMetricsEndpoint(server) {
```

Comment:
I don't think we should have this endpoint turned on in production. Could we remove it? Or put it under a config flag?

Reply:
This one is only for testing - it's easy to check metrics in influx format (we can pull them instead of pushing it somewhere). I don't have a strong need to save it. Maybe it's not worth to maintain it. I will remove it.

Reply:
done